packages/x-components/tests/e2e/cucumber/tagging/tagging.spec.ts

```diff
@@ -21,17 +21,28 @@ When('first redirection is clicked', () => {
 });
 
 When('scrolls down to next page', () => {
-  cy.getByDataTest('result-link').last().scrollIntoView();
+  cy.getByDataTest('result-link').last().scrollIntoView({ easing: 'linear', duration: 2000 });
 });
 
 Then('result click tagging request is triggered', () => {
-  cy.wait('@clickTagging');
+  cy.get('@clickTagging').should('exist');
 });
 
-Then('query tagging request is triggered', () => {
+Then('query tagging request should be triggered', () => {
   cy.wait('@queryTagging');
 });
 
+Then('query tagging request has been triggered', () => {
```

Comment:
This assertion is talking about the past, when the tagging request should happen immediately, therefore the `get`